comodo2/src/comodo2/queries/QTransition.java

```diff
@@ -191,6 +222,16 @@ public boolean isTimerTransition(final Transition t) {
 		return mQEvent.isTimeEvent(t.getTriggers().get(0).getEvent());
 	}
 
+	/** 
+	 * Used for QPC target, instead of isTimerTransition
+	 */
+	public boolean isTimerTransition_v2(final Transition t) {
```

Comment:
isTimerTransition could be renamed to isTimerTransitionWithEvent()
isTimerTransition_v2() could be renamed to isTimerTransition()  and the description updated to avoid target platform
(Actually I will have to update all documentation at one point ... ).